### DISCUSSION TITLE
One week until Next.js Conf. What do you hope to see?

### DISCUSSION THREAD

**Discussion by leerob** [vercel/next.js#56998] [Polls]
As you might have heard us on the Next.js team talk about, we've been focusing on performance, reliability, and further stability over the past months. We're now one week away from https://nextjs.org/conf and we have some great updates to share with you.

What are you hoping we talk about? I think a couple options in the poll but please comment with more.

**P.S.** You're going to _love_ the community talks! The best lineup we've ever had, if I do say so myself.

**Comment by icyJoseph** [comment]
I have been quite busy at work, so I might be out-of-date right now, but I'd like to hear/know a bit more about the current state of `Unit Testing` with React Server Components, when it comes to a server tree that uses async components.

**Reply by leerob** [reply]
Definitely need more docs here, and also this RFC is interesting: https://github.com/vercel/next.js/discussions/53409

**Comment by pawelblaszczyk5** [comment]
I’d like to see some stabilization around mutations/actions and cache APIs. Especially the current `unstable_cache`.

**Reply by leerob** [reply]
We plan to talk about more Server Actions for sure. `unstable_cache` is not yet ready to be stabilized, but we are open to feedback.

**Reply by willviles** [reply]
`unstable_cache` is the biggie for me. I don't want to drop tRPC, because of its ease of data validation with zod and ability to reuse endpoint logic in client components with its nice react-query data loading APIs. 

However, to benefit from request caching, I've had to duplicate all my server logic in API endpoints and use `fetch`.

Big headache, which would be resolved with `unstable_cache` becoming prod ready.

**Comment by lodybo** [comment]
I'm really looking forward to using Server Actions but we can't really use them until they're stable/production ready. So I'm hoping for some updates on that front..

**Reply by leerob** [reply]
You can expect updates here 🙏

**Reply by lodybo** [reply]
That was a nice surprise, thanks!
